docs/terminal/sydney.md

```diff
@@ -340,63 +349,72 @@ Any aircraft that don't meet these criteria must be coordinated to SY TCU with a
-    **SY TCU** -> **BK TWR**: "TFX12, Right Heading 290, A030"  
-    **BK TWR** -> **SY TCU**: "Right Heading 290, A030, TFX12"  
-    **BK TWR** Will then clear the aircraft to takeoff with the assigned heading, and instruct them to contact SY TCU passing A020.
+    **BK ADC** -> **SY TCU**: "Next, TFX12"  
+    **SY TCU** -> **BK ADC**: "TFX12, Right Heading 290, A030"  
+    **BK ADC** -> **SY TCU**: "Right Heading 290, A030, TFX12"  
+
+    **BK ADC** Will then clear the aircraft to takeoff with the assigned heading, and instruct them to contact SY TCU passing A020.  
+    `AIP GEN 3.4`
 
 #### Arrivals
 
-YSBK arrivals shall be coordinated to **BK TWR** from SY TCU prior to transfer of jurisdiction.
+YSBK arrivals shall be coordinated to **BK ADC** from SY TCU prior to transfer of jurisdiction.
 
 !!! tip
-    Ensure the aircraft's FDR is up-to-date in order to give **BK TWR** maximum situational awareness of the traffic picture. (eg. if the aircraft is doing the RNP approach, ensure the FDR has been rerouted via the appropriate points)
+    Ensure the aircraft's FDR is up-to-date in order to give **BK ADC** maximum situational awareness of the traffic picture. (eg. if the aircraft is doing the RNP approach, ensure the FDR has been rerouted via the appropriate points)
 
 !!! example
-    **SY TCU** -> **BK TWR**: "ABC, via ODALE, Number 1"  
-    **BK TWR** -> **SY TCU**: "ABC, Number 1"  
+    **SY TCU** -> **BK ADC**: "ABC, via ODALE, Number 1"  
+    **BK ADC** -> **SY TCU**: "ABC, Number 1"   
+    `AIP GEN 3.4`
 
-### SY TCU / CN TWR
+### SY TCU / CN ADC
 #### Departures
 
-Aircraft departing YSCN in to SY TCU Class C will be coordinated from **CN TWR** at Taxi.
+Aircraft departing YSCN in to SY TCU Class C will be coordinated from **CN ADC** at Taxi.
 
 !!! example
-    **CN TWR** -> **SY TCU**: "Taxi, DEF for YBTH via KADOM"  
-    **SY TCU** -> **CN TWR**: "DEF, Squawk 3601."  
-    **CN TWR** -> **SY TCU**: "Squawk 3601, DEF"  
+    **CN ADC** -> **SY TCU**: "Taxi, DEF for YBTH via KADOM"  
+    **SY TCU** -> **CN ADC**: "DEF, Squawk 3601."  
+    **CN ADC** -> **SY TCU**: "Squawk 3601, DEF"  
+    `AIP GEN 3.4`
 
-CN TWR will then give a "Next" call.
+CN ADC will then give a "Next" call.
 
 !!! example
-    **CN TWR** -> **SY TCU**: "Next, DEF"  
-    **SY TCU** -> **CN TWR**: "DEF"  
+    **CN ADC** -> **SY TCU**: "Next, DEF"  
+    **SY TCU** -> **CN ADC**: "DEF"  
+    `AIP GEN 3.4`
 #### Arrivals
 
-YSCN arrivals shall be coordinated to **CN TWR** from SY TCU prior to transfer of jurisdiction.
+YSCN arrivals shall be coordinated to **CN ADC** from SY TCU prior to transfer of jurisdiction.
 
 !!! example
-    **SY TCU** -> **CN TWR**: "New Sequence of 2, ZYX, via TONTO, Number 1. QET, via WATLE, Number 2"  
-    **CN TWR** -> **SY TCU**: "ZYX Number 1, QET Number 2"  
-### SY TCU / RIC TWR
+    **SY TCU** -> **CN ADC**: "New Sequence of 2, ZYX, via TONTO, Number 1. QET, via WATLE, Number 2"  
```

Comment:
Remove Sequence Number.
Might as well change this example to a single aircraft, not back to back.
"Estimate, ZYX via RNP W at Time XX" "RNP W ZYX"

Reply:
will require some other pages to be changed. maybe a future pull